rspamd.service

```diff
@@ -9,9 +9,39 @@ NonBlocking=true
 ExecStart=/usr/bin/rspamd -c /etc/rspamd/rspamd.conf -f
 ExecReload=/bin/kill -HUP $MAINPID
 User=_rspamd
+ConfigurationDirectory=rspamd
 RuntimeDirectory=rspamd
 RuntimeDirectoryMode=0755
+StateDirectory=rspamd
+StateDirectoryMode=0700
+LogsDirectory=rspamd
 Restart=always
 
+AmbientCapabilities=
+CapabilityBoundingSet=
+DevicePolicy=closed
+LockPersonality=true
+NoNewPrivileges=true
+PrivateDevices=true
+PrivateMounts=true
+PrivateTmp=true
+PrivateUsers=true
```

Comment:
Creates a new usernamespace where only the rspamd and root exits. Everything else is mapped to "nobody".